pkg/api/authz.go

```diff
@@ -287,17 +282,35 @@ func AuthzHandler(ctlr *Controller) mux.MiddlewareFunc {
 
 			ctx := acCtrlr.getContext(acCtx, request)
 
-			/* Notes:
-			 	- since we only do READ actions in extensions, we can bypass authz for them
-				only need to know the username, whether the user is an admin, or what repos he can read.
-				let each extension to apply	authorization on them using localCtx.AccessControlContext{}
-			*/
-			if isExtensionURI(request.RequestURI) {
-				next.ServeHTTP(response, request.WithContext(ctx)) //nolint:contextcheck
+			next.ServeHTTP(response, request.WithContext(ctx)) //nolint:contextcheck
```

Comment:
Make sure we document what the extension writer needs to do in order to implement authorization in the new extension.